skimage/feature/scattering.py

```diff
@@ -0,0 +1,286 @@
+""" Scattering transform code licensed under
+    BSD license.
+    Original author: Sira Ferradans, based on a code developed by
+    Ivan Dokmanic and Michael Eickenberg
+"""
+import numpy as np
+from skimage._shared.utils import assert_nD
+from skimage.filters.filter_bank import multiresolution_filter_bank_morlet2d
+import warnings
+
+
+def _apply_fourier_mult(signals, filters):
+    """Spatial subsampling on the last two dimensions of X at a
```

Comment:
The function name really doesn't make it sound like it is doing spatial subsampling. And looking at the code it is not :) So maybe you can say in the first sentence that this is multiplication of two tensors in Fourier space with the goal to obtain a convolution in signal space.